database/datagenerator.js

```diff
@@ -0,0 +1,15 @@
+var faker = require('faker');
```

Comment:
did you set up eslint? Check your json.package and made sure estlint is there and npm i. It'll help clean up your code!